lib.php

```diff
@@ -51,25 +51,30 @@ function accredible_add_instance($post) {
     $usersclient = new users();
     $accredible = new accredible();
 
-    // Load grade attributes for users who will get a credential issued if need to be added.
-    $userids = array();
-    foreach ($post->users as $userid => $issuecertificate) {
-        if ($issuecertificate) {
-            $userids[] = $userid;
-        }
-    }
-    $gradeattributes = $usersclient->get_user_grades($post, $userids);
+    $recordid = $accredible->save_record($post);
```

Comment:
Saving the post first since the created $record is necessary to generate custom attributes for manually issuing credentials.